Makefile

```diff
@@ -14,6 +14,7 @@ BASE=$(GOPATH)/src/$(REPO_PATH)
 GOFILES = $(shell find . -name *.go | grep -vE "(\/vendor\/)|(_test.go)")
 PKGS     = $(or $(PKG),$(shell cd $(BASE) && env GOPATH=$(GOPATH) $(GO) list ./... | grep -v "^$(PACKAGE)/vendor/"))
 TESTPKGS = $(shell env GOPATH=$(GOPATH) $(GO) list -f '{{ if or .TestGoFiles .XTestGoFiles }}{{ .ImportPath }}{{ end }}' $(PKGS))
+CONTAINER_ENGINE ?= $(shell /bin/bash -c "if command -v podman &>/dev/null; then echo podman; else echo docker; fi")
```

Comment:
IMAGE_BUILDER ?= docker

I would prefer something like this, check if the env var is set, if so, take the value, otherwise default.
I would prefer docker to be the default as well.

WDYT?

Reply:
done: https://github.com/k8snetworkplumbingwg/sriov-cni/pull/276/commits/fdbccf1feb606d58661debca58dab505550dc143